hack/go-tools/update-gofmt.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eou pipefail
+
+find . -name "*.go" | grep -v "\/vendor\/" | xargs gofmt -s -w
```

Comment:
This should support running against the root of the repo regardless from where the script is executed i.e. `.` is insufficient here. See the other scripts for an example such as `verify-gofmt.sh`.